forecasters/animalia/R/normalize_by_population.R

```diff
@@ -0,0 +1,54 @@
+#' Helper function for normalizing COVIDcast signals by population
+#' 
+#' Helper function to normalize COVIDcast signals by population. This replaces
+#' the `value` column in each signal with the appropriately normalized
+#' value. The user can choose which signals to normalize. Currently works with
+#' "state" and "county" `geo_type` only.
+#' 
+#' @param df_list List of signal values to use for forecasting, of the format
+#'   that is returned by [covidcast::covidcast_signals()].
+#' @param norm_by_popn Should the response/features be normalized by population?
+#'   This can be a single boolean value or a list of boolean values having the
+#'   same length as the number of elements in the `df` list, which tells us
+#'   which signals in `df` should be normalized by population and which
+#'   should not. Default is `FALSE`, i.e. no normalization.
+#' @param base The base for the normalization, i.e. the resulting values 
+#' should be interpreted as `value` per `base`. Default is `1e5`.
+#' 
+#' @return `df_list` but with the `value` column appropriately normalized.
+#' 
+#' @importFrom dplyr mutate select
+normalize_by_population <- function(df_list, norm_by_popn, base = 1e5) {
```

Comment:
let's go ahead and export this function.

also, `norm_by_popn` is not a particularly meaningful name. How about `signals_to_normalize` or some such?

I'd also like to give it default values. I think you can do: `signals_to_normalize = rep(list(FALSE), nsigs)`. It will get evaluated lazily after `nsigs` is computed, and also avoids some error checking.

Reply:
Sure. I missed that it would handle a length one logical. Can it deal with a vector of logicals rather than a list?